.github/dependabot.yml

```diff
@@ -7,6 +7,7 @@ updates:
     directory: "/"
     schedule:
       interval: "weekly"
+    open-pull-requests-limit: 10
```

Comment:
https://docs.github.com/en/code-security/dependabot/dependabot-version-updates/configuration-options-for-the-dependabot.yml-file#open-pull-requests-limit

> _**By default, Dependabot opens a maximum of five pull requests for version updates.** Once there are five open pull requests from Dependabot, Dependabot will not open any new requests until some of those open requests are merged or closed._